src/views/SystemList/Table.tsx

```diff
@@ -44,16 +79,22 @@ export const Table = ({ systems, savedBookmarks }: TableProps) => {
           const rowOneElem = rowOne.values[columnName];
           const rowTwoElem = rowTwo.values[columnName];
 
-          return findBookmark(rowOneElem, savedBookmarks) >
-            findBookmark(rowTwoElem, savedBookmarks)
+          return bookmarkIdSet.has(rowOneElem) > bookmarkIdSet.has(rowTwoElem)
```

Comment:
I think this can be clarified a bit, using `>` on two booleans isn't quite obvious. The [documentation](https://react-table.tanstack.com/docs/api/useSortBy#column-options) says "The sortType function should return 1 if rowA is larger, and -1 if rowB is larger."; given that, I think the appropriate expression would just be `bookmarkIdSet.has(rowOneElem) ? 1 : -1`. Thoughts?

Reply:
That makes sense. I think this particular code change was the just me removing a parameter from every call of findBookmark all at once.